apps/examples/board_specific_demo/ameba_wifi_csi_demo/example_wifi_csi.c

```diff
@@ -0,0 +1,126 @@
+/******************************************************************************
+ *
+ * Copyright(c) 2007 - 2015 Realtek Corporation. All rights reserved.
+ *
+ *
+ ******************************************************************************/
+
+
+#include <stdio.h>
+#include <tinyara/wifi/rtk/wifi_structures.h>
+#include "example_wifi_csi.h"
+
+
+#define IP_ADDR_INVALID 0x00000000
+int getIp(void);
+extern int wifi_csi_config(rtw_csi_action_parm_t *act_param);
+extern int wifi_csi_report(uint32_t buf_len, uint8_t *csi_buf, uint32_t *len, rtw_csi_header_t *csi_header);
+extern int wifi_is_running(unsigned char wlan_idx);
+sem_t wc_ready_sema;

```

Comment:
How about adding `static`?

Reply:
No change and no reply. Could you let us know your opinion for this?
This variable is only used in this file.

Reply:
Dear @sunghan-chang,
Are you referring to sem_t wc_ready_sema? I will add static for it.

Reply:
Correct.
We suggest adding static.
```
static sem_t wc_ready_sema;
```